src/main/java/org/beta/authentification/keycloak/authenticator/OtpLoginAuthenticatorFactory.java

```diff
@@ -39,12 +40,24 @@ public class OtpLoginAuthenticatorFactory implements AuthenticatorFactory {
 
     @Override
     public Authenticator create(KeycloakSession session) {
+        String tchapEmail = Environment.getenv(Constants.TCHAP_BOT_ACCOUNT_EMAIL);
```

Comment:
Constants.TCHAP_BOT_ACCOUNT_EMAIL -> would that be more meaningful to name "TCHAP_BOT_ACCOUNT_EMAIL_DEFAULT" to enhance the fact that it is a default value

Reply:
Definetely, but I think it's less cumbersome for the next release to let it in place like that, in order to reduce friction during first deployement of this new console.

Reply:
ok